packages/dom/src/dom.js

```diff
@@ -638,13 +633,15 @@ export function unwrap( node ) {
 	parent.removeChild( node );
 }
 
+// eslint-disable-next-line valid-jsdoc
```

Comment:
I assume this will need to be updated, since we no longer use `valid-jsdoc` but rather `eslint-plugin-jsdoc` (which has its own validity rule).

Reply:
Yeah sorry about that. What's left to be done here? Forgot I had this PR open. (We can close this if it's too far behind)

Reply:
> Yeah sorry about that. What's left to be done here? Forgot I had this PR open. (We can close this if it's too far behind)

I was going to ask the same 😄 It seems in pretty good shape otherwise, it would be good to get in. This one caught my eye though. There could also be some other more-recent changes to account for. Could do for a rebase.